inst/rmarkdown/templates/tuftish_ebook/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,276 @@
+---
+title: "Tuftish e-Book"
+author: "John Smith"
+date: "February 8, 2015"
+bibliography: "knitcitations.bib"
+output: 
+  tuftish_ebook:
+    latex_engine: xelatex
```

Comment:
Let's remove the latex_engine argument here (too in the weeds for an example and will just encourage people to change it to pdflatex!)